docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -473,9 +493,11 @@ Content-Type: text/plain;charset=UTF-8
 user%
 ----
 
-== Security JPA Reference Guide 
+:sectnums!:
 
-Now that you have run and tested the demo, please have a look at the more specific information about preparing your JPA identity store.
+== JPA security information 
```

Comment:
```suggestion
== Security JPA Reference Guide
```

Reply:
@sberyozkin .. guidance for titles is here: https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/doc-reference.adoc#titles-and-headings

Reply:
Hi @michelle-purcell This is perfect, thank you :-)